lib/Data/Chronicle/Subscriber.pm

```diff
@@ -0,0 +1,168 @@
+package Data::Chronicle::Subscriber;
+
+use 5.014;
+use strict;
+use warnings;
+use Data::Chronicle;
+use Date::Utility;
+use JSON::MaybeUTF8 qw(encode_json_utf8);
+use Moose;
+
+=head1 NAME
+
+Data::Chronicle::Subscriber - Provides callback subscriptions to an efficient data storage for volatile and time-based data
+
+=cut
+
+## VERSION
+
+=head1 DESCRIPTION
+
+This module contains helper methods which can be used to store and retrieve information
+on an efficient storage with below properties:
+
+=over 4
+
+=item B<Timeliness>
+
+It is assumed that data to be stored are time-based meaning they change over time and the latest version is most important for us.
+
+=item B<Efficient>
+
+The module uses Redis cache to provide efficient data storage and retrieval.
+
+=item B<Persistent>
+
+In addition to caching every incoming data, it is also stored in PostgreSQL for future retrieval.
+
+=item B<Transparent>
+
+This modules hides all the details about distribution, caching, database structure and ... from developer. He only needs to call a method
```

Comment:
"and ..." seems odd wording in documentation. Could you be more specific?

Reply:
I didn't write that, I assume the original author just meant `etc`.
I'll re-word it.